components/script/dom/htmlmediaelement.rs

```diff
@@ -1376,6 +1379,22 @@ impl HTMLMediaElementMethods for HTMLMediaElement {
     fn Played(&self) -> DomRoot<TimeRanges> {
         TimeRanges::new(self.global().as_window(), self.played.clone())
     }
+
+    // https://html.spec.whatwg.org/multipage/#dom-media-volume
+    fn GetVolume(&self) -> Fallible<Finite<f64>> {
+        Ok(Finite::wrap(self.volume.get()))
+    }
+
+    // https://html.spec.whatwg.org/multipage/#dom-media-volume
+    fn SetVolume(&self, value: Finite<f64>) -> ErrorResult {
+        let minimum_volume = 0.0;
+        let maximum_volume = 1.0;
+        if *value < minimum_volume || *value > maximum_volume {
+            return Err(Error::IndexSize);
```

Comment:
This return isn't throwing the error.

 > UNEXPECTED-PASS Check if media.volume is set to new value less than 0.0 that expecting an IndexSizeError exception is to be thrown

any suggestion?

Reply:
UNEXPECTED-PASS means that the ini file says the test currently fails on master, but with your changes it is passing. That sounds like it is doing the right thing and the ini file needs to be updated.

Reply:
yeah, Sorry, You are right, I miss understand it for a while